.github/workflows/lgtm.yaml

```diff
@@ -0,0 +1,16 @@
+name: "Merge on lgtm label"
+on:
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: "0 * * * *"
+
+jobs:
```

Comment:
@jamiemonserrate @anusha94 Do we require Auto Merge?

Reply:
Again, I'm nervous about it. No reason. Just nervous 😨

Reply:
Ok, let me clarify a few points that will clear your doubts.

1. The `/lgtm` label does not do anything on its own, it just signals that you've finished reviewing a PR and are satisfied with the changes. No merge will take place when you say `/lgtm` unless you approve the PR.
2. Once the reviewer is satisfied with changes, they can say `/approve`. If they want one more eye to go through the changes they can request another reviewer.
3. The best part is, we have branch protections enabled for PR merge. What does it mean, unless all the review comments are resolved, and the PR got 2 approvals, none of these labels will take precedence.
4. As the owner and reviewer of the code repository, we never blindly approve PR, we take precautions and go through the changes carfully, we only approve when we think everything will work fine (some exceptions, because we are human :))
5. The `auto-merge` will only be enabled if you click on the `Enable auto-merge(squash)` button as shown in this PR. This means that once all the criteria(test passes, all comments resolved, 2 approvals) to merge the PR is fulfiled, the PR will merge after that and you don't have to come again to merge the PR manually.

Please let me know if you want more details.

Reply:
Convinced! +1 🙂